packages/quantic/force-app/main/default/lwc/quanticSourceCitations/quanticSourceCitations.html

```diff
@@ -0,0 +1,40 @@
+<template>
+  <template if:true={shouldDisplayCitations}>
+    <div class="source__citations slds-grid slds-align_vertical-center">
+      <span
+        class="slds-shrink-none slds-m-top_xx-small slds-grid slds-align_absolute-center"
+        >{labels.learnMore}</span
+      >
```

Comment:
As it can be seen in the mockups, the text color of the `learn more` text shouldn't be black, it should be `#747474` instead.
or we can use the `slds-text-color_weak` slds class with it instead to stay more aligned with the Salesforce styles.